packages/application-components/src/components/tabular-detail-page/tabular-detail-page.tsx

```diff
@@ -29,7 +30,7 @@ type TTabularDetailPageProps = {
   /**
    * The title of the page.
    */
-  title: string;
+  title?: string;
```

Comment:
Can you also update the docs (prop types table) to make this optional?

We don't have this automated in appkit yet. Thanks